packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts

```diff
@@ -685,7 +704,7 @@ export default class TableResize implements EditorPlugin {
         });
     };
 
-    private resizeCells = (e: MouseEvent) => {
+    public resizeCells = (e: MouseEvent) => {
```

Comment:
Yes, it is used by test code and that's the tricky part...because we are using runAsync for resizeCells here, even we can trigger it to execute, we are not able to test it since we cannot immediate get the result.

Reply:
you can mock editor.runAsync() to let it do sync call. There are a lot of such examples, just search runAsync from test code

Reply:
working now!